src/api/api.cc

```diff
@@ -10338,12 +10338,20 @@ void recordreplay::BeginDisallowEvents() {
   }
 }
 
+extern "C" void V8RecordReplayBeginDisallowEvents() {
```

Comment:
These aren't being used anywhere?

Reply:
https://github.com/replayio/chromium/pull/170 uses these -- chromium's record_replay.h already has an AutoDisallowEvents definition but the methods it calls weren't actually defined so we were getting link errors when it is used.